services/app-web/src/pages/StateSubmission/SubmissionType/SubmissionType.tsx

```diff
@@ -155,7 +155,13 @@ export const SubmissionType = ({
 
     const handleFormSubmit = async (
         values: SubmissionTypeFormValues,
-        formikHelpers: FormikHelpers<SubmissionTypeFormValues>
+        formikHelpers: Pick<
+            FormikHelpers<SubmissionTypeFormValues>,
+            'setSubmitting'
+        >,
+        options?: {
+            redirectPath?: string
+        }
```

Comment:
Added the `options.redirectPath` to push to custom path ( '/dashboard' ) after  `updateDraft` mutation

Reply:
That looks good can you complete the refactor then -  only use `redirectPath`. You can remove the if/else and pass in the contract-details path for cases when we are redirecting there.